presto-base-jdbc/src/test/java/io/prestosql/plugin/jdbc/TestJdbcQueryBuilder.java

```diff
@@ -363,6 +365,22 @@ public void testBuildSqlWithTimestamp()
         }
     }
 
+    @Test
+    public void testBuildSqlWithLimit()
+            throws SQLException
+    {
+        Connection connection = database.getConnection();
+        Function<String, String> function = sql -> format("SELECT * FROM (%s) LIMIT 10", sql);
```

Comment:
Why not just `"%s LIMIT 10"` ?

Reply:
Good catch, this is leftover from earlier testing. Updated